shared/validation/place-order-request-validator.ts

```diff
@@ -94,8 +95,23 @@ function isShopOpen(date: Date): boolean {
     return true;
 }
 
+function isMinimumOrderSatisfied(order: PlaceOrderRequestValidationObject): string[] {
+    let error: string[] = [];
+    if (order.deliveryMethod === 'Collection' && order.paymentMethod === 'cash') {
+        return error;
+    }
+    let priceOfPizzaItems = order.orderItems.filter(x => x.tags && x.tags.indexOf('pizza') > -1)
```

Comment:
Once you add tags to all items, you can get rid of `x.tags && `

A small stylistic comment. It's more common to see `!== -1` as opposed to `> -1`